### DISCUSSION TITLE
parallel-group-params-mismatch but params match across machines

### DISCUSSION THREAD

**Discussion by mgoode87** [cypress-io/cypress#17114] [CI setup]
Recently we have been running into an issue with one of our cypress jenkins jobs, it consistently fails on  "parallel-group-params-mismatch" but the params match across the machines.   We tried limiting the number of concurrent jobs also to see if that would make a difference but we still receive the same error.     

The logs shows this for both machines and they are identical: 
`You passed the --parallel flag, but we do not parallelize tests across different environments.

This machine is sending different environment parameters than the first machine that started this parallel run.

The existing run is: https://dashboard.cypress.io/projects/project-id/runs

In order to run in parallel mode each machine must send identical environment parameters such as:

- specs
- osName
- osVersion
- browserName
- browserVersion (major)

This machine sent the following parameters:

{
 ​"osName": "linux",
 ​"osVersion": "Debian - 10.9",
 ​"browserName": "Chrome",
 ​"browserVersion": "90.0.4430.212",
 ​"specs": [...]
}

https://on.cypress.io/parallel-group-params-mismatch`

I found this [issue](https://github.com/cypress-io/github-action/issues/121) from 2020 but its not too helpful in our situation since our browser versions match.   

Anyone have any ideas what could be causing our jobs to fail on this error?

**Comment by bmulholland** [comment]
I'm getting flaky tests today with this error. Stock Github Actions.
